packages/services/api/src/modules/rate-limit/resolvers.ts

```diff
@@ -1,5 +1,5 @@
-import { Logger } from './../shared/providers/logger';
 import { RateLimitModule } from './__generated__/types';
+import { Logger } from './../shared/providers/logger';
 import { RateLimitProvider } from './providers/rate-limit.provider';
```

Comment:
```suggestion
import { RateLimitModule } from './__generated__/types';
import { Logger } from './../shared/providers/logger';
import { RateLimitProvider } from './providers/rate-limit.provider';
```

I guess here we need apply ESLint rule to suggest do not use `./` in path :)

Reply:
I agree!

Reply:
Lovely!!